Bugzilla/Template.pm

```diff
@@ -398,7 +398,13 @@ sub css_files {
 
     unshift @requested_css, "skins/yui.css" unless $no_yui;
 
-    my @css_sets = map { _css_link_set($_) } @requested_css;
+    my @css_sets = map { _css_link_set($_) } sort {
+        my $first_a = $a =~ m!js/jquery!;
+        my $first_b = $b =~ m!js/jquery!;
```

Comment:
In the case when the regex doesn't match, the regex is faster than index().
Though with the number of elements (and size of the strings) it doesn't matter either way.